### PR TITLE
Update MySQL Docs

### PR DESCRIPTION
**Pull request by simonaco**
Update example with ES6 syntax.

### CODE REVIEW

File: docs-ref-services/mysql.md [hunk 1]
```diff
@@ -31,23 +31,22 @@ npm install mysql
...
+  database: 'my_db'
 });
 
 connection.connect();
-
-connection.query('SELECT 1 + 1 AS solution', function (error, results, fields) {
-  if (error) throw error;
-  console.log('The solution is: ', results[0].solution);
-});
+const query = 'SELECT 1 + 1 AS solution';
```
Comment (johnpapa) [inline docs-ref-services/mysql.md]: I like that this runs without any tables, but I think it is OK here to assume the reader will be more comfortable with a basic query like

`SELECT * FROM customers`

If you prefer, add some text to the above sentence that says 

> Connect to a MySQL database and perform a simple query to get all customers

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/mysql.md](https://github.com/simonaco/azure-docs-sdk-node/blob/patch-1/docs-ref-services/mysql.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C13%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707131235440146-79%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/mysql.md [hunk 2]
```diff
@@ -31,23 +31,22 @@ npm install mysql
...
+connection.query(query, (err, res) =>
+  console.log('The solution is: ', res[0].solution)
```
Comment (johnpapa) [inline docs-ref-services/mysql.md]: Consider something with the template string syntax

```
console.log(`We have ${res.length} customers`)
```

just an example.

File: docs-ref-services/mysql.md [hunk 3]
```diff
@@ -13,15 +13,15 @@ ms.devlang: nodejs
...
+# Azure MySQL packages for Node.js
 
 ## Overview
 
 The recommended client library for accessing Azure Database for MySQL is the open-source [Node.js connection library for Azure Database for MySQL](https://github.com/sidorares/node-mysql2/tree/master/documentation). 
 
 Learn more about [Azure Database for MySQL](https://docs.microsoft.com/azure/MySQL/)
 
-## Install modules with npm
+## Install packages with npm
```
Comment (johnpapa) [inline docs-ref-services/mysql.md]: ## Management Package
2 #'s

### Install npm package
3 #'s

File: docs-ref-services/mysql.md [hunk 4]
```diff
@@ -31,23 +31,22 @@ npm install mysql
 
 ## Example
 
-Connect to, and query, a MySQL database.
+Connect to a MySQL database and perform a simple query.
```
Comment (johnpapa) [inline docs-ref-services/mysql.md]: 3 #'s for Example

File: docs-ref-services/mysql.md [hunk 5]
```diff
@@ -31,23 +31,22 @@ npm install mysql
...
 connection.connect();
-
-connection.query('SELECT 1 + 1 AS solution', function (error, results, fields) {
-  if (error) throw error;
-  console.log('The solution is: ', results[0].solution);
-});
+const query = 'SELECT 1 + 1 AS solution';
+connection.query(query, (err, res) =>
+  console.log('The solution is: ', res[0].solution)
+);
```
Comment (johnpapa) [inline docs-ref-services/mysql.md]: Make `Samples` be 3 #'s

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/mysql.md](https://github.com/simonaco/azure-docs-sdk-node/blob/patch-1/docs-ref-services/mysql.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C14%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707141009220536-79%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/mysql.md](https://github.com/simonaco/azure-docs-sdk-node/blob/patch-1/docs-ref-services/mysql.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C14%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707141237460857-79%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by simonaco** [comment]
#sign-off

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/mysql.md](https://github.com/simonaco/azure-docs-sdk-node/blob/patch-1/docs-ref-services/mysql.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C14%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707141245528871-79%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by TomArcher** [comment]
@simonaco Merged

**Pull request merged**